.github/workflows/build.yml

```diff
@@ -2,10 +2,18 @@ name: "SQS UI | Build Docker Image"
 
 on:
   workflow_dispatch:
+    inputs:
+      docker_tag:
+        description: 'Docker tag to build'
+        required: false
+        default: ''
   push:
-    branches: [ "main", "v*.*.*" ]
-  pull_request:
-    branches: [ "main", "v*.*.*" ]
+    tags:
+      - sqs-v[0-9]+.[0-9]+.[0-9]+
+    branches:
+      - "v[0-9]**"
```

Comment:
I think that the old one is correct but not the new one. E.g. `v21.x` does not work